smart-contracts/contracts/mixins/MixinRefunds.sol

```diff
@@ -38,6 +38,27 @@ contract MixinRefunds is
     uint refundPenaltyDenominator
   );
 
+  /**
+   * @dev Invoked by the lock owner to destroy the user's ket and perform a refund and cancellation
+   * of the key
+   */
+  function fullRefund(address _keyOwner, uint amount)
```

Comment:
In a future PR I'll refactor to see if we can combine with the logic of user trigger refunds.